### PR TITLE
Adds the GAIA benchark to the Testbed. This PR depends on #792

### PR DESCRIPTION
**Pull request by afourney**
## Why are these changes needed?

This PR adds initial support for the GAIA benchmark to the Testbed. 

**Note**: This PR depends on #792. Merge that one first.

## Related issue number

N/A

## Checks

- [ ] I've included any doc changes needed for https://microsoft.github.io/autogen/. See https://microsoft.github.io/autogen/docs/Contribute#documentation to build and test documentation locally.
- [ ] I've added tests (if relevant) corresponding to the changes introduced in this PR.
- [ ] I've made sure all auto checks have passed.

### CODE REVIEW

**Review by qingyun-wu — APPROVED** [review]
LGTM! Following the practice here, should @LeoLjl move utils for AutoGPT to the utils folder? if so, @LeoLjl could you do that in a separate PR? Thank you!

File: samples/tools/testbed/scenarios/GAIA/Templates/BasicTwoAgents/scenario.py [hunk 1]
```diff
@@ -0,0 +1,66 @@
...
+
+testbed_utils.init()
+##############################
+
+
+GAIA_SYSTEM_MESSAGE = (
+    "You are a helpful AI assistant, and today's date is "
+    + datetime.now().date().isoformat()
+    + """.
+I will ask you a question. Answer this quesiton using your coding and language skills.
```
Comment (LeoLjl) [inline samples/tools/testbed/scenarios/GAIA/Templates/BasicTwoAgents/scenario.py:15]:
```suggestion
I will ask you a question. Answer this question using your coding and language skills.
```

**Review by LeoLjl — APPROVED** [review]
LGTM

**Comment by LeoLjl** [comment]
> LGTM! Following the practice here, should @LeoLjl move utils for AutoGPT to the utils folder? if so, @LeoLjl could you do that in a separate PR? Thank you!

@qingyun-wu I'll copy the utils to the utils folder. Will do it in a separate PR.

**Pull request merged**